Modules/README.md

```diff
@@ -9,5 +9,6 @@ sys-netcheat | netcheat sys-module by jakibaki
 edizon | EdiZon
 appstore | AppstoreNX
 Checkpoint | Save File Manager
-sdfilesupdater | The SDFilesUpdater homebrew
-hbmenu | The homebrew menu
\ No newline at end of file
+kosmossupdater | The Kosmos homebrew
+hbmenu | The homebrew menu
+kosmos_toolkit | Homebrew for setup your cfw settings
```

Comment:
"Homebrew for modifying CFW settings" would be correct english

Reply:
https://github.com/AtlasNX/Kosmos/pull/156/commits/8a1112dc3c1668f4351150bfba034d6ac25fbf4d